lib/public/views/Runs/Overview/RunsOverviewModel.js

```diff
@@ -100,10 +100,26 @@ export class RunsOverviewModel extends OverviewPageModel {
         const updateDebounceTime = () => {
             this._debouncedLoad = debounce(this.load.bind(this), model.inputDebounceTime);
         };
+
```

Comment:
A bit higher in the code you have: `this._allRuns`
Is this still needed now that you have extracted the export functionality in its own model and component? I think it was being used in the previous component`exportRunsTriggerAndModal`